op-challenger/game/fault/trace/vm/kona_interop_server_executor.go

```diff
@@ -0,0 +1,48 @@
+package vm
+
+import (
+	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/trace/utils"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type KonaInteropExecutor struct {
+	nativeMode     bool
+	agreedPreState []byte
+}
+
+var _ OracleServerExecutor = (*KonaInteropExecutor)(nil)
+
+func NewKonaInteropExecutor(agreedPreState []byte) *KonaInteropExecutor {
+	return &KonaInteropExecutor{nativeMode: false, agreedPreState: agreedPreState}
+}
+
+func NewNativeKonaInteropExecutor(agreedPreState []byte) *KonaInteropExecutor {
+	return &KonaInteropExecutor{nativeMode: true, agreedPreState: agreedPreState}
+}
+
+func (s *KonaInteropExecutor) OracleCommand(cfg Config, dataDir string, inputs utils.LocalGameInputs) ([]string, error) {
```

Comment:
We'll eventually want to expand `utils.LocalGameInputs` to contain the pre-state preimage, but I imagine this'll come along with other challenger work. LMK what your plans are here @ajsutton - this works for now for the sake of the action tests.

Reply:
Oh good point, yes it needs to be added to `LocalGameInputs`.  Feel free to do that here if you want - probably easier than having to remember to come back and change this executor again later.  It can be left unset for all cases challenger currently handles and then I can set it in https://github.com/ethereum-optimism/optimism/pull/13794/files#diff-c1954bb5f021bb8692515e052d1ba63bc6c587e2d3e876842b0a9519366cbd10